src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -289,7 +292,7 @@ private async Task<DebugLaunchSettings> GetConsoleTargetForProfile(ILaunchProfil
                     {
                         // Try to resolve against the current working directory (for compat) and failing that, the environment path.
                         string exeName = executable.EndsWith(".exe", StringComparisons.Paths) ? executable : executable + ".exe";
-                        string fullPath = _fileSystem.GetFullPath(exeName);
+                        string? fullPath = _fileSystem.GetFullPath(exeName);
```

Comment:
This will throw on null, not return null.

Reply:
The variable is reused on line 302 to store a nullable reference.

I'll introduce another nullable variable for that usage and keep this non-null.